etl/steps/export/explorers/minerals/latest/minerals_supply_and_demand_prospects.py

```diff
@@ -0,0 +1,159 @@
+    df_graphers["Metric Radio"] = metric_radio
+    df_graphers["Demand Scenario Dropdown"] = scenario_dropdown
+    df_graphers["Case Dropdown"] = case_dropdown
+    df_graphers["hasMapTab"] = map_tab
+
+    # Impose that all line charts start at zero.
+    df_graphers["yAxisMin"] = 0
+
+    # Make all views stacked area charts.
+    df_graphers["type"] = "StackedArea"
+
+    # Sanity check.
+    error = "Duplicated rows in explorer."
+    assert df_graphers[
+        df_graphers.duplicated(
+            subset=[
+                "Indicator Radio",
+                "Case Dropdown",
+                "Demand Scenario Dropdown",
+                "Mineral Dropdown",
+                "Type Dropdown",
+                "Metric Radio",
+            ],
+            keep=False,
+        )
+    ].empty, error
+
+    # Sort rows conveniently.
+    # To ensure categories appear in the right order, they corresponding column can be defined as categorical.
+    # NOTE: But ensure they are defined in the same way as they appear in the data, otherwise they will not appear in the explorer.
+    scenarios_sorted = ["All scenarios", "Stated policies", "Announced pledges", "Net zero by 2050"]
+    error = "The defined categories do not coincide with scenarios. They will not appear in the explorer."
+    assert set(df_graphers["Demand Scenario Dropdown"]) == set(scenarios_sorted), error
+    df_graphers["Demand Scenario Dropdown"] = pd.Categorical(
+        df_graphers["Demand Scenario Dropdown"],
+        categories=scenarios_sorted,
+        ordered=True,
+    )
+    # NOTE: Ideally, the "Type Dropdown" would be sorted as first "Mine" and then "Refinery" (for consistency with the
+    # minerals explorer). However, the first mineral in this explorer is "Arsenic", which only has "Refinery" type, and
+    # therefore, "Refinery" will always be the first choice in that dropdown.
+    df_graphers = df_graphers.sort_values(
+        [
+            "Indicator Radio",
+            "Mineral Dropdown",
+            "Type Dropdown",
+            "Demand Scenario Dropdown",
+            "Case Dropdown",
+            "Metric Radio",
+        ]
+    ).reset_index(drop=True)
+
+    # Choose which indicator to show by default when opening the explorer.
+    df_graphers["defaultView"] = False
+    default_mask = (
+        (df_graphers["Indicator Radio"] == "Demand by technology")
+        & (df_graphers["Mineral Dropdown"] == "Copper")
+        & (df_graphers["Type Dropdown"] == "Refinery")
+        & (df_graphers["Case Dropdown"] == "Base case")
+        & (df_graphers["Demand Scenario Dropdown"] == "Net zero by 2050")
+        & (df_graphers["Metric Radio"] == "Total")
+    )
+    assert len(df_graphers[default_mask]) == 1, "Multiple rows selected for default view."
+    df_graphers.loc[default_mask, "defaultView"] = True
+
+    # Prepare explorer metadata.
+    config = {
+        "explorerTitle": "Minerals Supply and Demand Prospects",
+        "explorerSubtitle": "",
```

Comment:
I think all explorers have a subtitle in the style of: _Explore data from xxx_, for them to be shown in https://ourworldindata.org/charts